tests/test_cli.py

```diff
@@ -33,6 +32,13 @@ def home_dir(tmp_path):
 
 @pytest.fixture
 def config_dir(home_dir):
+    d = home_dir / ".config/aocpy"
+    d.mkdir(parents=True)
+    return d
+
+
+@pytest.fixture
+def cache_dir(home_dir):
```

Comment:
These *should* be the same, `.config/aocd` is a mistake now I look at it. The cached files really ought to be stored separately from the config in something like `~/.cache/aocpy` however that's probably a separate issue. No need to change your PR, I'll fix this up later.